MAVProxy/modules/mavproxy_ws.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+'''
+WebSocket Module
+Jim Heising
+August 2024
+'''
+
+from MAVProxy.modules.lib import mp_module, mp_settings
+from websockets.sync.server import serve
+from threading import Thread
+from websockets import ConnectionClosedOK
+from pymavlink import mavutil
+import json
+import traceback
+
+mavlink_map = mavutil.mavlink.mavlink_map
+message_name_2_type_cache = {}
+
+# Some examples of JSON MAVLink messages
+# {"mavpackettype": "COMMAND_LONG", "target_system":0, "target_component": 0, "command": 400, "confirmation": 0, "param1": 1, "param2": 0, "param3": 0, "param4": 0, "param5": 0, "param6": 0, "param7": 0}
```

Comment:
You really want the header in here too. So you can check where the messages are coming from. There is already a "standard" for MAVLink over JSON from the mavlink2rest lib here: https://github.com/mavlink/mavlink2rest?tab=readme-ov-file#api

Reply:
That's a great callout. I will update the PR to move to that.